src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -19,13 +19,14 @@ If you're more adventurous or want to get the latest changes of the xk6-browser
 
 <InstallationInstructions extensionUrl="github.com/grafana/xk6-browser"/>
 
-## Launch
+## Launch Chromium
 
-The first step is to launch a [Browser](/javascript-api/xk6-browser/browser). After the browser starts, you can interact with it using the [browser-level APIs](#browser-level-apis). The only accepted browser type is currently `chromium`.
+The first step is to launch a `chromium` [Browser](/javascript-api/xk6-browser/browser) (which is currently the only available browser type). After the browser starts, you can interact with it using the [browser-level APIs](#browser-level-apis).
 
-| Method                         | Description                                                                                                                                                                                                         |
-|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| launch(browserName, [options]) | The first parameter is required, and the only accepted `browserName` is currently `chromium`. The `options` parameter is optional. You can see a list of all possible `options` in [the options heading](#options). |
+| Method                     | Description                                                                      |
+|----------------------------|----------------------------------------------------------------------------------|
+| chromium                   | Import module from `'k6/x/browser'`                                              |
+| chromium.launch([options]) | You can see a list of all possible `options` in [the options heading](#options). |
```

Comment:
It's a bit strange listing `chromium` under "Method". Technically, `import { chromium } from 'k6/x/browser';` should import the Chromium `BrowserType` implementation/object directly, and we should remove [our `launch()` wrapper that currently exists at the module level](https://github.com/grafana/xk6-browser/blob/aadaffec87c8ffba66db89f417bb68e3aacbbe08/main.go#L71-L87).

So we should probably add a new page for `BrowserType`, where we document `launch()` and all of its options. And on this page we should document the module itself, which has global attributes like `devices` and `version`.

I'll create issues for both of these things, and hopefully we'll have time to work on them in this cycle. They shouldn't take long.